admin/includes/modules/copy_product_confirm.php

```diff
@@ -145,11 +145,11 @@
                                              WHERE products_id = '" . $products_id . "'");
 
             $db->Execute("UPDATE " . TABLE_PRODUCTS . " SET
-                metatags_title_status = '" . zen_db_input($metatags_status->fields['metatags_title_status']). "',
-                metatags_products_name_status = '" . zen_db_input($metatags_status->fields['metatags_products_name_status']). "',
-                metatags_model_status = '" . zen_db_input($metatags_status->fields['metatags_model_status']). "',
-                metatags_price_status= '" . zen_db_input($metatags_status->fields['metatags_price_status']). "',
-                metatags_title_tagline_status = '" . zen_db_input($metatags_status->fields['metatags_title_tagline_status']). "'
+                metatags_title_status = '" . zen_db_input($metatags_status->fields['metatags_title_status']) . "',
+                metatags_products_name_status = '" . zen_db_input($metatags_status->fields['metatags_products_name_status']) . "',
+                metatags_model_status = '" . zen_db_input($metatags_status->fields['metatags_model_status']) . "',
+                metatags_price_status= '" . zen_db_input($metatags_status->fields['metatags_price_status']) . "',
+                metatags_title_tagline_status = '" . zen_db_input($metatags_status->fields['metatags_title_tagline_status']) . "'
```

Comment:
Aren't all of these integer values? Shouldn't `zen_db_input` basically be replaced with `(int)` and the single quotes removed from around the value (in addition to standardizing the text to have the additional space before the period and the extra parentheses removed from around the variable associated with `zen_db_input`)?

Reply:
Sure. But immaterial to this PR.